.github/workflows/pages.yml

```diff
@@ -0,0 +1,66 @@
+# Sample workflow for building and deploying a Hugo site to GitHub Pages
```

Comment:
We already have a github pages workflow right.. so is this file added by some accident?